### ISSUE TITLE
How do I run the randomizer program?

### ISSUE THREAD

**Issue by CoolGaming14** [RoccoDev/botw-randomizer#1]
So I have no idea what to do. I put my Botw Wii U rom into the folder and tried running it with the start-windows.bat and also the gradlew.bat files but nothing happens. What am I doing wrong?

**Comment by codemann8** [comment]
Yeah @RoccoDev I've been looking at this for the past 4 days. I have no clue what I need to be doing to get started on this. I also tried running javac RandomizerMain but there were errors there as well. A little description that outlines how to actually use and run this would be very good to include in the Readme.
